### ISSUE TITLE
Using ray/ beam spells through dispensers crashes the game when targeting players blocking with a shield. 

### ISSUE THREAD

**Issue by NoMoreUsernames999** [Electroblob77/Wizardry#717]
Minecraft version: 1.12.2 
Wizardry version: 4.3.4 
Environment: Singleplayer

Issue details: Using any type of beam or ray spell through a dispenser on a player who is blocking with a shield causes a crash. Tested with lightning ray, fire breath, ice ray, life drain, and wither. The crash is always the same. Yet to test with players VS shield instead of dispensers, but that would be a huge problem with having to disable all beam type spells on servers :( 

Other mods involved: None. The crash is identical with spartan shields and vanilla shields so it seems to be a problem with Wizardry and the vanilla damage blocking mechanic.

Link to crash report: https://pastebin.pl/view/e891d96e

**Comment by WinDanesz** [comment]
If I recall correctly this crash also happens if you die from a dispenser's spell ray. Without using a shield

**Comment by NoMoreUsernames999** [comment]
You're right it does. So this means the problem is likely with the dispensers and not with the spells since I don't remember this happening with players. Would have been nice to have a config option that disables dispenser casting, since that would have been a temporary fix. I guess I'll just have to delay server launch until Electroblob gets it fixed :/

**Comment by WinDanesz** [comment]
You can disable dispenser casting for each spell. You just need to duplicate their .json files in your server's config folder, there are more details on that in the wiki https://github.com/Electroblob77/Wizardry/wiki/Spell-Properties

**Comment by WinDanesz** [comment]
Actually this link is more relevant https://github.com/Electroblob77/Wizardry/wiki/Spell-Properties

**Comment by WinDanesz** [comment]
All jsons go to the `config/ebwizardry/spell` folder. What you need to do, is set `"dispensers": false` for each

**Comment by WinDanesz** [comment]
[spells.zip](https://github.com/Electroblob77/Wizardry/files/7206510/spells.zip)
if this helps - here is a copy of all spell jsons with dispensers disabled for all of them

**Comment by NoMoreUsernames999** [comment]
Oh nice, I thought I remembered a way to disable them. Was looking at the client side config. Thanks again Dan. This saved me a lot of time.
